ext/hash/tests/hash_update_stream_basic_001.phpt

```diff
@@ -0,0 +1,39 @@
+--TEST--
+Ensure hash_update_stream() always returns the same hash when $length = 0
+--CREDITS--
+Symeon Charalabides <symeon@systasis.com> - @phpdublin
+--SKIPIF--
+<?php
+if (phpversion() < "5.3.0") {
+    die('SKIP - php version too low.');
+}
+if (!extension_loaded('hash')) die('skip hash extension not available');
```

Comment:
I'm not sure about this either, as the Hash extension cannot be unbundled as of PHP 7.4, so if this is targeting master this will always fail.
But I don't know what the policy is for that in all honesty.

Reply:
The respective SKIPIF clauses for other test cases are removed as of PHP 7.4.0, so indeed that line above can simply be removed.

Reply:
This is an old pull request. Fixed now.